pkg/sysfs/system.go

```diff
@@ -104,6 +104,7 @@ type System interface {
 	CoreKindCPUs(CoreKind) cpuset.CPUSet
 	CoreKinds() []CoreKind
 	AllThreadsForCPUs(cpuset.CPUSet) cpuset.CPUSet
+	SingleThreadPerCore(cpuset.CPUSet) cpuset.CPUSet
```

Comment:
nitpick: although sysfs is already far from being anywhere consistent wrt. naming, I'd still prefer either misnaming this to `SingleThreadForCPUs` or renaming the existing `AllThreadsForCPUs` to `AllThreads[For|Per]Cores`.

Reply:
Renamed to SingleThreadForCPUs.